internal/rest/avi_obj_vs.go

```diff
@@ -607,6 +607,9 @@ func (rest *RestOperations) AviVsCacheAdd(rest_op *utils.RestOp, key string) err
 		if found_parent {
 			// the uuid is expected to be in the format: "https://IP:PORT/api/virtualservice/virtualservice-88fd9718-f4f9-4e2b-9552-d31336330e0e#mygateway"
 			vs_uuid := avicache.ExtractUuid(vh_parent_uuid.(string), "virtualservice-.*.#")
+			if !lib.AKOControlConfig().IsLeader() {
```

Comment:
comment from code review: Not required once we add the `include_name=true` to the path for GET operations.

Reply:
Addressed